images/themes/default/mActionRemoveItemPoint.svg

```diff
@@ -0,0 +1,332 @@
/zGgM6vxHpNBDpAL3EOenEOXArfU4DfQ7Q S5+TTp/zFgM6rxLpNBDpAH3EOenEOXAvfU4DfQ7QR5+TTp/zHgM6bxLpNBDpAD3EOenEOfAofU4D fQ7QQ5+TTp/zEQM67xLpNBDpAPnEOenEOfAsfU4DfQ6QT5+TTp9zCwM6HxLpNBDpALnEOenEObAX fU4DfQ6QS5+TTp9zKwM6NxHpNBDpAHnEOenEObA3fU4DfQ6QR5+TTp9zDwM6NxPpNBDpADnEOenE OXAUfU4DfQ6QQ5+TTp9zLwM6dxHpNBDpAPMT56QT58DR9DkN9DnA/PQ56fQ5jzCgczeRTgORDjAv cU46cQ6cRZ/TQJ8DzEufk06f8ygDOg8R6TQQ6QDzEeekE+fA2fQ5DfQ5wHz0Oen0Oc8woPMwkU4D kQ4wD3FOOnEOjKLPaaDPAeahz0mnz3mWAZ2niHQaiHSA8cQ56cQ5MJo+p4E+BxhPn5NOn7MHAzpP E+k0EOkA44hz0olzYBb6nAb6HGAcfU46fc5eDOjsQqTTQKQDnE+ck06cA7PR5zTQ5wDn0+ek0+fs yYDObkQ6DUQ6wHnEOenEOTArfU4DfQ5wHn1OOn3O3gzo7Eqk00CkAxxPnJNOnAOz0+c00OcAx9Pn pNPnHMGAzu5EOg1EOsBxxDnpxDmQQp/TQJ8DHEefk06fcxQDOocQ6TQQ6QD7E+ekE+dAGn1OA30O sD99Tjp9zpEM6BxGpNNApAPsR5yTTpwDqfQ5DfQ5wH70Oen0OUczoHMokU4DkQ7wPHFOOnEOpNPn NNDnAM/T56TT55zBgM7hRDoNRDrA48Q56cQ50EKf00CfAzxOn5NOn3MWAzqnEOk0EOkA9xPnpBPn QBt9TgN9DnA/fU46fc6ZDOicRqTTQKQD3E6ck06cA630OQ30OcDt9Dnp9DlnM6BzKpFOA5EO8DFx TjpxDrTT5zTQ5wAf0+ek0+eMYEDndCKdBiId4G3inHTiHFiFPqeBPgd4mz4nnT5nFAM6Q4h0Goh0 gN+Jc9KJc2A1+pwG+hzgd/qcdPqckQzoDCPSaSDSAX4Q56QT58Cq9DkN9DnAD/qcdPqc0QzoDCXS aSDSAcQ5+cQ5sDp9TgN9DqDPyafPmYEBneFEOg1EOrAycU46cQ7wnT6ngT4HVqbPSafPmYUBnSmI dBqIdGBF4px04hzgV/qcBvocWJE+J50+ZyYGdKYh0mkg0oGViHPSiXOA1+lzGuhzYCX6nHT6nNkY 0JmKSKeBSAdWIM5JJ84B3qfPaaDPgRXoc9Lpc2ZkQGc6Ip0GIh1oJs5JJ84BbqPPaaDPgWb6nHT6 nFkZ0JmSSKeBSAcaiXPSiXOA++hzGuhzoJE+J50+Z2YGdKYl0mkg0oEm4px04hzgMfqcBvocaKLP SafPmZ0BnamJdBqIdKCBOCedOAd4jj6ngUYU5ZoAACAASURBVD4HGuhz0ulzEhjQmZ5Ip4FIB5KJ c9KJc4B96HMa6HMgmT4nnT4nhQGdCCKdBiIdSCTOSSfOAfalz2mgz4FE+px0+pwkBnRiiHQaiHQg iTgnnTgHOIY+p4E+B5Loc9Lpc9IY0Iki0mkg0oEE4px04hzgWPqcBvocSKDPSafPSWRAJ45Ip4FI B2YmzkknzgHOoc9poM+Bmelz0ulzUhnQiSTSaSDSgRmJc9KJc4Bz6XMa6HNgRvqcdPqcZAZ0Yol0 Goh0YCbinHTiHGAMfU4DfQ7MRJ+TTp+TzoBONJFOA5EOzECck06cA4ylz2mgz4EZ6HPS6XMaGNCJ J9JpINKBkcQ56cQ5wBz0OQ30OTCSPiedPqeFAZ0KIp0GIh0YQZyTTpwDzEWf00CfAyPoc9Lpc5oY 0Kkh0mkg0oEziXPSiXOAOelzGuhz4Ez6nHT6nDYGdKqIdBqIdOAM4px04hxgbvqcBvocOIM+J50+ p5EBnToinQYiHTiSOCedOAfIoM9poM+BI+lz0ulzWhnQqSTSaSDSgSOIc9KJc4As+pwG+hw4gj4n nT6nmQGdWiKdBiId2JM4J504B8ikz2mgz4E96XPS6XPaGdCpJtJpINKBPYhz0olzgGz6nAb6HNiD PiedPmcFBnTqiXQaiHTgGeKcdOIcoIM+p4E+B56hz0mnz1mFAZ0liHQaiHTgEeKcdOIcoIs+p4E+ Bx6hz0mnz1mJAZ1liHQaiHTgHuKcdOIcoJM+p4E+B+6hz0mnz1mNAZ2liHQaiHTgFuKcdOIcoJs+ p4E+B26hz0mnz1mRAZ3liHQaiHTgPeKcdOIcYA36nAb6HHiPPiedPmdVBnSWJNJpINKB14hz0olz gLXocxroc+A1+px0+pyVGdBZlkingUgHfibOSSfOAdakz2mgz4Gf6XPS6XNWZ0BnaSKdBiId2DZx Tj5xDrA2fU4DfQ5smz4nnz4HAzqIdCqIdFibOCedOAdg2/Q5HfQ5rE2fk06fw3cGdNhEOh1EOqxJ nJNOnAPwM31OA30Oa9LnpNPn8IMBHf5KpNNApMNaxDnpxDkAr9HnNNDnsBZ9Tjp9Dr8yoMNPRDoN RDqsQZyTTpwD8B59TgN9DmvQ56TT5/A7Azq8INJpINKhmzgnnTgH4Bb6nAb6HLrpc9Lpc3idAR1e IdJpINKhkzgnnTgH4B76nAb6HDrpc9Lpc3ibAR3eINJpINKhizgnnTgH4BH6nAb6HLroc9Lpc3if AR3eIdJpINKhgzgnnTgH4Bn6nAb6HDroc9Lpc/iYAR0+INJpINIhmzgnnTgHYA/6nAb6HLLpc9Lp c7iNAR1uINJpINIhkzgnnTgHYE/6nAb6HDLpc9Lpc7idAR1uJNJpINIhizgnnTgH4Aj6nAb6HLLo c9Lpc7iPAR3uINJpINIhgzgnnTgH4Ej6nAb6HDLoc9Lpc7ifAR3uJNJpINJhbuKcdOIcgDPocxro c5ibPiedPofHGNDhASKdBiId5iTOSSfOATiTPqe
```

Comment:
Embedding an image in the svg like this means it won't scale well. Can you vectorize this cross part of the icon?